package.json

```diff
@@ -21,5 +21,8 @@
 		"vercel:login": "yarn workspace web vercel login",
 		"vercel:link": "yarn workspace web vercel link",
 		"vercel:env:pull": "yarn workspace web vercel env pull"
+	},
+	"dependencies": {
+		"react-color": "^2.19.3"
```

Comment:
This should get moved into the `package.json` for the web package.

`yarn remove react-color && yarn workspace web add react-color`